legacy/definition.go

```diff
@@ -440,6 +440,17 @@ func migrateAction(baseLanguage utils.Language, a Action, localization flows.Loc
 			quickReplies = TransformTranslations(legacyQuickReplies)
 		}
 
+		for lang, attachment := range media {
+			parts := strings.SplitN(attachment, ":", 2)
+			mediaType := parts[0]
+			mediaURL := parts[1]
+
+			if strings.Contains(mediaType, "/") {
```

Comment:
Shouldn't we be looking for `@` ?

Reply:
this is differentiating between _foo/bar:relativeurl_ and _foo:template_ .. the thing about the latter is it doesn't necessarily contain a `@` expression. This matches how `ReplyAction` determines whether or not to add `STORAGE_URL` in the legacy engine.

Reply:
Ah, makes sense